### ISSUE TITLE
Military > Update Fleet info

### ISSUE THREAD

**Issue by melomelo7** [sparticle999/SpaceCompany#431]
Barely started to spawn stuff : 1 scout
sent it to do some spy job on 1st faction obviously it was lost
but point here is fleet info after lost : 
still shows 3 pow 2 def 15 spe and can perform action again so again lose the scout
( if lost should not be showing infos/action if not then modify text only ... anyways lets say bug ? )

**Comment by Rare-Account** [comment]
Yeah it’s a bug it fixes once you click on ex. “Espionage” and then set the number of scouts to 0 it fixes it
